src/Auth0Client.ts

```diff
@@ -311,6 +311,8 @@ export default class Auth0Client {
 
     this.useRefreshTokensFallback =
       this.options.useRefreshTokensFallback !== false;
+
+    this.options.useFormData = this.options.useFormData !== false;
```

Comment:
Should we update the object passed in as options or track a separate property to avoid mutating the object that's being passed in?